src/test/java/teammates/test/cases/datatransfer/CourseAttributesTest.java

```diff
@@ -17,7 +18,7 @@
 
     private String validName = "validName";
     private String validId = "validId";
-    private String validTimeZone = "validTimeZone";
+    private ZoneId validTimeZone = ZoneId.of("UTC");
```

Comment:
Same as above for `Const.DEFAULT_TIMEZONE`.

Reply:
This is a test. It should explicitly states what timezone it's using